sourcegraph/sourcegraph.proto

```diff
@@ -142,7 +146,9 @@ message Repo {
 	// Fork is whether this repository is a fork.
 	bool fork = 12;
 
-	// Mirror is whether this repository is a mirror.
+	// Mirror indicates whether this repo's canonical location is on
+	// another server. Mirror repos track their upstream and are not
+	// eligible for discovery on this server.
 	bool mirror = 13;
```

Comment:
You could mirror a repo that is hosted on another Sourcegraph instance. But it'd make more sense for you to just access it via federation. I suppose it could make sense if you wanted to have totally separate issues, comments, and other non-git data for the repo.